server.js

```diff
@@ -29,20 +30,47 @@ app.get('/static', (req, res) => {
     res.render('pages/static')
 })
 
+const stateStore = new Map()
```

Comment:
**future:** We'll want to add a comment here with caveats about how this isn't something you'll want to do in Production (i.e. it's only done here in favor of _simplicity_ of the example code).

I'm thinking something similar to what we did for the other example project a la https://github.com/Banno/consumer-api-openid-connect-example/blob/91f7e0febbc5d87fe33d37345c2d5520480138a2/server.js#L39-L42